pkg/puppetdb/nodes.go

```diff
@@ -2,15 +2,35 @@ package puppetdb
 
 import (
 	"fmt"
+	"strconv"
 )
 
+func (p Pagination) toParams() map[string]string {
```

Comment:
Params is generic to all the queries, so can move it's definition and methods into a separate file i.e. pkg/puppetdb/pagination.go.